src/test/java/org/heigit/ohsome/ohsomeapi/controller/DataExtractionTest.java

```diff
@@ -54,61 +54,62 @@ public static void applicationMainShutdown() {
-    assertEquals(7, feature.get("properties").size());
+    assertEquals(6, feature.get("properties").size());
   }
 
   @Test
   public void elementsGeomUsingOneTagTest() {
     TestRestTemplate restTemplate = new TestRestTemplate();
     ResponseEntity<JsonNode> response = restTemplate.getForEntity(server + port
-        + "/elements/geometry?bboxes=8.67452,49.40961,8.70392,49.41823&types=way&keys=building"
-        + "&values=residential&time=2015-12-01&properties=metadata", JsonNode.class);
+        + "/elements/geometry?bboxes=8.67452,49.40961,8.70392,49.41823&time=2015-12-01"
+        + "&properties=metadata&filter=type:way and building=residential", JsonNode.class);
     assertTrue(Helper.getFeatureByIdentifier(response, "@osmId", "way/140112811") != null);
   }
 
   @Test
   public void elementsGeomUsingMultipleTagsTest() {
     TestRestTemplate restTemplate = new TestRestTemplate();
-    ResponseEntity<JsonNode> response = restTemplate.getForEntity(
-        server + port
-            + "/elements/geometry?bboxes=8.67559,49.40853,8.69379,49.4231&types=way&keys=highway,"
-            + "name,maxspeed&values=residential&time=2015-10-01&properties=metadata",
+    ResponseEntity<JsonNode> response = restTemplate.getForEntity(server + port 
+        + "/elements/geometry?bboxes=8.67559,49.40853,8.69379,49.4231&time=2015-10-01"
+        + "&properties=metadata&filter=type:way and highway=residential and maxspeed=* and name=*",
         JsonNode.class);
     assertTrue(Helper.getFeatureByIdentifier(response, "@osmId", "way/4084860") != null);
   }
 
   @Test
   public void elementsGeomUnclippedSimpleFeaturesTest() {
     TestRestTemplate restTemplate = new TestRestTemplate();
-    ResponseEntity<JsonNode> response = restTemplate.getForEntity(
-        server + port
-            + "/elements/geometry?bboxes=8.700582,49.4143039,8.701247,49.414994&types=other,line&"
-            + "keys=building&showMetadata=true&properties=unclipped&time=2019-01-02",
+    ResponseEntity<JsonNode> response = restTemplate.getForEntity(server + port
+        + "/elements/geometry?bboxes=8.700582,49.4143039,8.701247,49.414994&properties=unclipped"
+        + "&time=2019-01-02&filter=building=* and (geometry:other or geometry:line)",
         JsonNode.class);
     assertTrue(response.getBody().get("features").size() == 0);
   }
 
-  @Test
-  public void elementsGeomSimpleFeaturesOtherLineTest() {
-    TestRestTemplate restTemplate = new TestRestTemplate();
-    ResponseEntity<JsonNode> response = restTemplate.getForEntity(server + port
-        + "/elements/geometry?bboxes=8.700582,49.4143039,8.701247,49.414994&types=other,line&"
-        + "keys=building&showMetadata=true&time=2019-01-02", JsonNode.class);
-    assertTrue("GeometryCollection"
-        .equals(response.getBody().get("features").get(0).get("geometry").get("type").asText()));
-  }
+  // this needs a fix in the OSHDB to work
+  // see https://github.com/GIScience/oshdb/issues/338
+//  @Test
```

Comment:
the indentation of this commented-out code block doesn't seem right:

https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/job/PR-100/14/checkstyle/fileName.-336662661/category.-641434199/source.ebc16fea-0e58-46b3-91e8-c37d929e8d48/#105

Reply:
I think it is ok to copy/paste a couple of tests and to change them using the old syntax. We can put them below so that they are easy to be found and deleted in the future.

Concerning the not-needed tests, a couple of big spatial meetings ago it was said that the whole tests should be revised, but this now was not the aim of this PR.

Reply:
> the whole tests should be revised

Ok, if this is in the pipeline than it's fine to leave the tests for now. But it would be good to create an issue for the tests-revision, so we don't forget about it.

> I think it is ok to copy/paste a couple of tests and to change them using the old syntax. We can put them below so that they are easy to be found and deleted in the future.

sounds good to me